src/openrct2/world/Map.cpp

```diff
@@ -1747,7 +1747,9 @@ static void clear_elements_at(const CoordsXY& loc)
     gPeepSpawns.erase(
         std::remove_if(
             gPeepSpawns.begin(), gPeepSpawns.end(),
-            [x = loc.x, y = loc.y](const auto& spawn) { return floor2(spawn.x, 32) == x && floor2(spawn.y, 32) == y; }),
+            [x = loc.x, y = loc.y](const auto& spawn) {
+                return spawn.ToTileStart().x == x && spawn.ToTileStart().y == y;
```

Comment:
If you can change line 1750 to take `const TileCoordsXY& spawn)` (I don't know if that compiles), then you could do `return spawn.ToTileStart() == loc;`

Reply:
...\OpenRCT2\src\openrct2\actions\FootpathRemoveAction.hpp(104,1): error C2039: 'ToTileStart': is not a member of 'TileCoordsXY' (compiling source file world\Map.cpp)

![image](https://user-images.githubusercontent.com/15891478/78066935-228a5080-7396-11ea-84c9-803fd22eea39.png)

Reply:
You should always try to avoid passing `this` into a lambda it rarely does what you expect. I think what @Gymnasiast meant to say was do it like this (also not tested for compiliation):

```
[loc](const CoordsXY& spawn){ return spawn.ToTileStart() == loc; }
```

Reply:
For the change in the map.cpp your code works fine. For the FootpathRemoveAction.hpp I can only get it to work with passing this (I guess it's because _loc is private?).

Reply:
I wrote TileCoordsXY where I meant to write CoordsXY. Sorry about that.